### DISCUSSION TITLE
Steps are no longer updating for me in Github Skills

### DISCUSSION THREAD

**Discussion by TheToastyAbyss** [skills/.github#249] [Q&A] [answered]
Hi all!

I was working on the resolve-merge-conflicts skills and believe I messed up somewhere cause I would refresh and it would not update. So I deleted that repository and now it will not update on any of the skills for me. Any Ideas on how to fix this? I'm assuming I messed something up when I deleted that repository.

**Comment by TheToastyAbyss** [comment, marked as answer]
UPDATE:

It is now working again. If someone did something to fix it thank you!

**Reply by sinsukehlab** [reply]
This is probably due to the latest GitHub internal incident.
See: https://www.githubstatus.com/incidents/1g1gkh0qpyvs

**Comment by DaiseyPaisley** [comment]
> Hi all!
> 
> I was working on the resolve-merge-conflicts skills and believe I messed up somewhere cause I would refresh and it would not update. So I deleted that repository and now it will not update on any of the skills for me. Any Ideas on how to fix this? I'm assuming I messed something up when I deleted that repository.

>Did you ever get This fixed? It still won't let me get last committing a file :(
